homepage/homepage/app/(docs)/docs/[framework]/[...slug]/faq.mdx

```diff
@@ -0,0 +1,18 @@
+# Frequently Asked Questions
+
+## What makes Jazz a good long-term choice?
+
+We're committed to Jazz being around for a long time! We understand that when developers choose Jazz for their personal projects, they're investing their time and creativity into us, and we take that responsibility seriously.
+That's why we've designed Jazz with longevity in mind from the start:
+
+- The open source nature of our sync server means you'll always be able to run your own infrastructure
+- Your data remains accessible even if our cloud services change
+- We're designing the protocol as an open specification
+
+This approach creates a foundation that can continue regardless of any single company's involvement. The local-first architecture means your apps will always work, even offline, and your data remains yours.
+
+## How established is Jazz?
+
+Jazz is backed by some great investors and has been in development [since 2023](https://github.com/garden-co/jazz/commit/d0993189d20fe78ec8b09abc3acfcf3ab738bd5d). We spent significant time researching local-first solutions, and finding the space lacking, we created Jazz. We're committed to building Jazz as a long-term, sustainable platform for local-first applications.
```

Comment:
```suggestion
Jazz is backed by fantastic angel and institutional investors with experience and know-how in devtools and has been in development since 2020. We spent significant time researching local-first solutions, and finding the space lacking, we created Jazz. We're committed to building Jazz as a long-term, sustainable platform for local-first applications.
```